core/templates/dev/head/services/contextual/UrlServiceSpec.ts

```diff
@@ -46,267 +35,267 @@ describe('Url Service', function() {
       search: ''
     };
 
-    UrlService = $injector.get('UrlService');
-    spyOn(UrlService, 'getCurrentLocation').and.returnValue(mockLocation);
-  }));
+    urlService = TestBed.get(UrlService);
+    spyOn(urlService, 'getCurrentLocation').and.returnValue(mockLocation);
+  });
 
   it('should return correct query value list for each query field', function() {
```

Comment:
All migrated frontend tests are moving from `function() {` to `() => {`

Reply:
Done...Noted!